mock/api.js

```diff
@@ -13,34 +13,19 @@ export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index)
   results: casual.integer(1, DEFAULT_SIZE),
 }));
 
-export const mockResults = {
-  took: DEFAULT_SIZE,
-  hits: {
-    total: {
-      value: DEFAULT_SIZE,
-    },
-    hits: new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-      _index: moment().format('YYYY-MM'),
-      _type: casual.word,
-      _id: casual.uuid,
-      _score: null,
-      _source: {
-        '@metadata': {
-          controller_dir: casual.word,
-        },
-        run: {
-          controller: casual.word + index,
-          name: casual.word,
-          start: moment.utc() + index,
-          end: moment.utc() + index,
-          id: casual.uuid,
-          config: casual.word,
-        },
-      },
-      sort: [casual.unix_time],
-    })),
-  },
-};
+export const mockResults = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
+  '@metadata.controller_dir': casual.word,
+  key: casual.uuid,
+  id: casual.uuid,
```

Comment:
Hmm. `casual.uuid` returns standard UUID 8-4-4-4-12 format, whereas the dataset `id` is the Elasticsearch document ID, which is the MD5 hash of the tarball... a long hex string like `3646953262b0adfe6d1c32903912087b`. Again, I don't think the dashboard code cares what it represents, so this is a nit; but mocking data that looks something like the real data is a better strategy. I don't think that `casual` has the generator we really want here, and UUID isn't likely to break anything. Maybe all it needs is a comment, or maybe removing the `-` characters.

Reply:
Replaced '-' characters in uuid to make it look like real data